sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1975,7 +1975,9 @@ Removing this restriction is something we may look at for future revisions of
 
 The command submission functions defined in
 link:../experimental/sycl_ext_oneapi_enqueue_functions.asciidoc[sycl_ext_oneapi_enqueue_functions]
-can be used to add nodes to a graph when creating a graph from queue recording.
+can be used adding nodes to a graph when creating a graph from queue recording
+and also submitting an executable graph e.g. to a queue directly without
+returning an event.
```

Comment:
Sentence feels a bit long, slight preference to keeping the existing wording then adding a new sentence.
```quote
New methods are also defined that enable submitting an executable graph, e.g directly to a queue without returning an event.  
```